src/constants.ts

```diff
@@ -23,4 +23,54 @@ export const TOOLTIP_COLOR = '#3B3649';
 export const SIDE_PANEL_WIDTH: number = 280;
 export const MIN_WIDTH_TO_SHOW_SIDE_PANEL: number = 580;
 export const VIEWER_INITIAL_WIDTH: number = 500;
-export const VIEWER_HEIGHT: number = 529;
+export const VIEWER_HEIGHT: number = 580;
+
+//todo get from viewer
```

Comment:
why is this a TODO? I'm pretty sure we export these from the viewer because we also use this on the front end

Reply:
Only UIDisplayEntry is exported, not DisplayStateEntry, so I think to use that directly we'd need to add it? I can also remove these both for now, they are holdovers from the larger set of code changes and it might not be needed.